src/styles/header.scss

```diff
@@ -0,0 +1,77 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05%);
+}
+
+.is-active {
+  position: relative;
+  padding: 23px 0;
+  font-size: 12px;
+  margin-right: 20px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  cursor: pointer;
+  font-family: inherit;
+  transition: color 0.3s ;
+  font-weight: 500;
+
+  &::after {
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: "";
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    margin-bottom: 0;
+  }
+}
+
+.nav {
```

Comment:
Move to separate file.

Reply:
It's a BEM block - so it should be moved to separate file